app/src/main/java/org/mozilla/focus/searchsuggestions/ui/SearchSuggestionsFragment.kt

```diff
@@ -0,0 +1,205 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.searchsuggestions.ui
+
+import android.arch.lifecycle.Observer
+import android.arch.lifecycle.ViewModelProviders
+import android.graphics.Color
+import android.graphics.drawable.BitmapDrawable
+import android.os.Bundle
+import android.support.v4.app.Fragment
+import android.support.v4.content.ContextCompat
+import android.support.v7.widget.LinearLayoutManager
+import android.support.v7.widget.RecyclerView
+import android.text.SpannableString
+import android.text.SpannableStringBuilder
+import android.text.Spanned
+import android.text.TextPaint
+import android.text.method.LinkMovementMethod
+import android.text.style.ClickableSpan
+import android.text.style.ForegroundColorSpan
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import kotlinx.android.synthetic.main.fragment_search_suggestions.*
```

Comment:
\[detekt\] INFO: \[style/WildcardImport\] Wildcard imports should be replaced with imports using fully qualified class names\. Wildcard imports can lead to naming conflicts\. A library update can introduce naming clashes with your classes which results in compilation errors\.

Reply:
@pocmo is this one new for the synthetic imports? We use wildcard imports everywhere else

Reply:
Yes, ignore. Sputnik config is crap :(